src/components/Objects/Input/Input.jsx

```diff
@@ -14,11 +14,9 @@ export default function Input({
 
   return (
     <label className={className ? `${className}-input-container` : 'input-container'}>
-        <div className='label-text-container'>
-          <h6>{label}</h6>
-          {errorMessage && <span>{errorMessage}</span>}
-        </div>
-        <input {...rest} aria-label={`${label}-input`} type={type} onChange={onValueChange}/>
+        <h6>{label}</h6>
+        <input className={errorMessage ? 'error-input' : ''} {...rest} aria-label={`${label}-input`} type={type} onChange={onValueChange}/>
+        {errorMessage && <span>{errorMessage}</span>}
```

Comment:
nah u can grab the tag inside the container rather than having to set a class name. Since it's the only span I do it like so

```
    & span {
        color: $olErrorMessage;
        font-size: 0.85rem;
        margin-left: 2%;
        margin-left: 4%;
        margin-top: 1%;
    }
```